crt_portal/cts_forms/tests/test_filters.py

```diff
@@ -217,7 +218,8 @@ def test_malformed_parens_search(self):
 
 
 class ReportLanguageFilterTests(TestCase):
-    def setUp(self):
+    @classmethod
+    def setUpTestData(self):
```

Comment:
@billyhunt Here is an end-of-day status update.  Tests continue to fail due to the duplicate key value violating the primary key restraint on `cts_forms_report_pkey`.  Output below:

```
======================================================================
ERROR: test_reported_language_vi (cts_forms.tests.test_filters.ReportLanguageFilterTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "cts_forms_report_pkey"
DETAIL:  Key (id)=(1) already exists.
```

I will investigate more on Monday.